.gitignore

```diff
@@ -280,4 +280,5 @@ __pycache__/
 /EventsExpress/Properties/PublishProfiles/*
 /EventsExpress/ClientApp/build
 /EventsExpress/.config/dotnet-tools.json 
-/.vscode/*
\ No newline at end of file
+/.vscode/*
+/EventsExpress.Test/GraphQLTests/__snapshots__/EventTests.GetEvents.snap
```

Comment:
Remove the folder and not individual files please.

Reply:
Done